docs/nGraph_DG/nGraph_Python_API.md

```diff
@@ -0,0 +1,222 @@
+# Using nGraph's Python API
+
+nGraph is OpenVINO's graph manipulation library, used to represent neural network models in the form of a computational graph. Using nGraph's Python APIs, you can create, inspect and manipulate computational graphs. You can also use the API to set runtime properties of the graph, which will be used by OpenVINO during inference.
+
+## nGraph namespace module
+
+nGraph has a large set of mathematical operations, chosen to cover most ops used in popular neural network models. The `ngraph` module is the basic namespace used to expose commonly used API methods. You can inspect the `ngraph` module or call `help` on any method for usage information.
+
+```python
+import ngraph as ng
+dir(ng)
+help(ng.space_to_batch)
+```
+
+## Create a simple graph
+
+nGraph's Python API can be used to create computational models. Let's say you want to create a model which represents the following formula:
```

Comment:
You can use nGraph Python API to create computational models. For example, to create a model that represents the following formula y=ReLU(x + b), you can use the following code to create parameters `x` and `b` and create the computational graph: